packages/embeds/embed-core/src/embed.ts

```diff
@@ -415,8 +415,17 @@ class CalApi {
   }
 
   init(namespaceOrConfig?: string | InitArgConfig, config = {} as InitArgConfig) {
+    let namespace = "";
     if (typeof namespaceOrConfig !== "string") {
       config = (namespaceOrConfig || {}) as Config;
+    } else {
+      namespace = namespaceOrConfig;
+    }
+
+    if (!globalCal.ns[namespace].instance && namespace !== this.cal.namespace) {
```

Comment:
If a namespace encounters the instruction for some other namespace, create an instance for that namespace and that would handle related work.

Ideally this should happen with default namespace encountering other namespace instruction. We can explicitly confirm that here I think.